recipes/oras-py/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "oras-py" %}
+{% set version = "0.0.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/oras-project/oras-py/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 3a1c991e122435cc3ae080fbdb0f7addcb75b17a5ec8afa22f87afc55dc93b7b
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - oras-py=oras.cli:run
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+    - pip
+    - jsonschema
+    - requests
+    - docker-py ==5.0.1
+    - pytest-runner
+    - pytest
+    - mypy
+    - pyflakes
+    - black
+    - types-requests
+    - isort
```

Comment:
Unless the package imports itself in the `setup.py` you don't need the runtime dependencies here. Even if that is true you probably do not need all the test ones. BTW, you should report this upstream b/c it is bad practice to require everything at install time.

```suggestion
    - pip
```